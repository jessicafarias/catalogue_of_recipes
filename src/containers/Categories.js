const Categories = () => (
  <div>
    {/* Mapping to the list to display component */}
  </div>
);

export default Categories;
