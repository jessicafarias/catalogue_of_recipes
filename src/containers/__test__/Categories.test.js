import { BrowserRouter } from 'react-router-dom';
import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer'; // ES6
import { Provider } from 'react-redux';
import { applyMiddleware, createStore } from 'redux';
import thunk from 'redux-thunk';
import Categories from '../Categories';
import rootReducer from '../../reducers';

const store = createStore(rootReducer, applyMiddleware(thunk));

describe('Display Categories.js', () => {
  it('renders correctly', () => {
    const tree = TestRenderer.create(
      <Provider store={store}>
        <BrowserRouter>
          <Categories />
        </BrowserRouter>
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
