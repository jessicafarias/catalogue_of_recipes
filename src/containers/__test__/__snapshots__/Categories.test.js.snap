// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Categories.js renders correctly 1`] = `
<div
  className="Categories h-100"
>
  <div
    className="container_filter sticky-top"
  >
    <label
      htmlFor="w-100"
    >
      <select
        className="select"
        id="select"
        name="category"
        onChange={[Function]}
      >
        <option
          disable="true"
          hidden={true}
        >
          Search any keyword
        </option>
        <option
          value="All"
        >
          All
        </option>
        <option
          value="Beef"
        >
          Beef
        </option>
        <option
          value="Breakfast"
        >
          Breakfast
        </option>
        <option
          value="Chicken"
        >
          Chicken
        </option>
        <option
          value="Dessert"
        >
          Dessert
        </option>
        <option
          value="Goat"
        >
          Goat
        </option>
        <option
          value="Lamb"
        >
          Lamb
        </option>
        <option
          value="Miscellaneous"
        >
          Miscellaneous
        </option>
        <option
          value="Pasta"
        >
          Pasta
        </option>
        <option
          value="Pork"
        >
          Pork
        </option>
        <option
          value="Seafood"
        >
          Seafood
        </option>
        <option
          value="Side"
        >
          Side
        </option>
        <option
          value="Starter"
        >
          Starter
        </option>
        <option
          value="Vegan"
        >
          Vegan
        </option>
        <option
          value="Vegetarian"
        >
          Vegetarian
        </option>
      </select>
    </label>
  </div>
  <img
    alt="meal"
    src="https://images.pexels.com/photos/1528013/pexels-photo-1528013.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260"
  />
</div>
`;
